Update automated-patching.md (#9534)
MicrosoftDocs/sql-docs: azure-sql/virtual-machines/windows/automated-patching.md

Update-AzSqlVM -ResourceGroupName 'resourcegroupname' -Name 'vmname' -AutoPatchingSettingEnable:$false
```

## Understanding What Updates will be applied with Automated Patching

> Please ensure to review the [Update Guide](https://msrc.microsoft.com/update-guide) and apply the **Severity** filter to identify Critical and Important updates in order to understand the updates that will be applied through Automated Patching.

Reviewer: ```suggestion
## Understand which updates will be applied with Automated Patching
To understand which updates will be applied through Automated Patching, review the [update guide](https://msrc.microsoft.com/update-guide) and apply the **Severity** filter to identify Critical and Important updates.
```
Minor edits for Microsoft style
